src/server/views/dashboard/users-list.njk

```diff
@@ -16,10 +23,12 @@
           <th scope="row" class="govuk-table__header">{{ user.email }}</th>
           <td class="govuk-table__cell">{{ user.jsonData.roles.join(", ") }}</td>
           <td class="govuk-table__cell govuk-table__cell--numeric">
+            {% if not user.jsonData.roles.includes('SuperAdmin') %}
```

Comment:
nothing for a superAdmin to do on another superAdmin user. Obscuring edit (but is also validated on server if they try to jump there)